src/components/views/rooms/RoomHeader.js

```diff
@@ -110,7 +110,7 @@ module.exports = React.createClass({
                 var searchStatus;
                 // don't display the search count until the search completes and
                 // gives us a non-null searchCount.
-                if (this.props.searchInfo && this.props.searchInfo.searchCount !== null) {
+                if (this.props.searchInfo && this.props.searchInfo.searchCount != null) {
```

Comment:
Presumably this is done to catch `0`? This will also catch `undefined` now. I generally don't like doing negative tests like this because it's not always clear what you really mean, which I'm *guessing* is "there is a search count". If so, I'd really prefer it if you just did that in the code via:

```javascript
if (this.props.searchInfo && this.props.searchInfo.searchCount) {
```

which will return true for truthy values (not 0, undefined or null).